archive-recipe.php

```diff
@@ -47,8 +47,17 @@
             $ingredients = get_terms($taxonomies, $args);
             if ($ingredients) {
               foreach ($ingredients  as $ingredient) {
+                $ingredient_name_en = get_term_meta( $ingredient->term_id, '_igv_ingredient_name_en', true );
             ?>
-              <li class="taxonomy-ingredient"><a href="<?php echo get_term_link( $ingredient ); ?>"><?php echo $ingredient->name; ?></a></li>
+              <li class="taxonomy-ingredient"><a href="<?php echo get_term_link( $ingredient ); ?>">
+                <?php 
+                  if (qtranxf_getLanguage() == 'en' && $ingredient_name_en) {
```

Comment:
that does callback to es brother. its asking if the current language is english AND an english name for the ingredient exists, print the english name, otherwise use the spanish name in all other cases. thats es fallback